src/index.js

```diff
@@ -29,7 +29,7 @@ const getMyScores = async (link) => {
 
```

Comment:
Since I can leave a comment directly on your README file, I will leave that comments in this section 😄
- - [ ] First of all, the link that takes to your `MIT` license is not working, kindly fix this, you can either erase that section of your README file or add an MIT license. 🥇 
- - [ ] Your live version of the project does not match the version I see when I start it locally.
    - This is your live version:
    ![imagen](https://user-images.githubusercontent.com/60789317/143623094-fc6626ea-cebc-4fd1-81af-09ad804ec9b2.png)
    - And this is the local version:
    ![imagen](https://user-images.githubusercontent.com/60789317/143623042-6c0ad214-668f-4486-90cf-0db3a04084d6.png)

Reply:
Thanks for the review!
I planned to add the right link once the project will be ready for deployment. but, let me do it right now as you ask me!
Thank you so far for the review!